job_definitions/create_index.yml

```diff
@@ -0,0 +1,56 @@
+{% for environment in ['preview', 'staging', 'production'] %}
+- job:
+    name: "create-index-{{ environment }}"
+    display-name: "Create index - {{ environment }}"
+    project-type: freestyle
+    description: >
+      This job creates a new index in the ElasticSearch service from specified objects in the database.
+
+      Visiting either https://search-api.<STAGE>.marketplace.team/_status or
+      https://search-api.digitalmarketplace.service.gov.uk/_status will show you currently available indexes and their
+      aliases.
+
+      You can use the above to get an idea of naming conventions.
+
+      Once you have created your new index you might want to use it on the site. To do so, you can give your new index
+      an 'alias' here https://ci.marketplace.team/job/update-{{ environment }}-index-alias/
+    parameters:
+      - choice:
+          name: OBJECTS
+          choices:
+            - briefs
+            - services
+          default: services
+      - string:
+          name: INDEX_NAME
+          default: {{ search_config['services'][environment].default_index }}
+          description: "A new name for your new index (eg 'g-cloud-9-2017-05-22')"
+      - string:
+          name: FRAMEWORKS
+          default: {{ search_config['services'][environment].frameworks }}
+          description: "Comma-separated list of framework slugs that should be indexed (eg 'g-cloud-7,g-cloud-8'). If no frameworks are specified then all currently published services will be indexed."
+      - string:
+          name: MAPPING
+          description: >
+            This mapping is a filename (without .json suffix) which should exist in the
+            digitalmarketplace-search-api/mappings directory
+            (https://github.com/alphagov/digitalmarketplace-search-api/tree/master/mappings)
+            (eg 'briefs-digital-outcomes-and-specialists')
+    triggers:
+      - timed: "H 2 * * *"
```

Comment:
Will this create a new index every morning? I think we just want to update indexes regularly overnight, not create new ones.